gateway-api/deploy.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Let's use starter-kit approach here (build code into libs and include a new docker image) instead of `sh` scripts.